lib/KListWithOverflow.vue

```diff
@@ -276,5 +294,8 @@
   .more-button-wrapper {
     visibility: hidden;
   }
+  .more-button-wrapper.start-button {
```

Comment:
We cant have this solution for the position of the button, because Keyboard navigation would not be the same. We need to keep the current behaviour in KBreadcrumbs that if the more button appears at the beginning, the keyboard navigation should focus it first, before the rest items.

https://github.com/user-attachments/assets/dbf2abb9-4a9e-471b-be0f-eb2081e959f3